src/irr/video/COpenGLPipelineCache.h

```diff
@@ -48,13 +49,25 @@ class COpenGLPipelineCache final : public IGPUPipelineCache
 		return {0,nullptr};
 	}
 	//assumes that m_cache does not already contain an item with key==_key and layout fully compatible with _val.layout
-	void insert(SCacheKey&& _key, SCacheVal&& _val)
+	void insert(SCacheKey&& _key, SCacheVal&& _val, const asset::ICPUBuffer* _spirv)
```

Comment:
you're caching the wrong thing here

Reply:
hm?

Reply:
we shouldn't be caching the SPIR-V shader, just the hash of the shader instead.

Reply:
yea, yea, ofc it does. This last param is just wrong approach to something else, doesnt matter, sorry. Will rework it today

Reply:
ok.